src/so_vits_svc_fork/preprocess_resample.py

```diff
@@ -43,9 +44,10 @@ def preprocess_one(input_path: Path, output_path: Path) -> None:
         soundfile.write(output_path, audio, samplerate=sampling_rate, subtype="PCM_16")
```

Comment:
please enclose process_one in try-catch and use rglob("**/*. *") accommodate more than just flac instead of using itertools.chain.